released/packages/coq-vst-32/coq-vst-32.2.9.1/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+synopsis: "Verified Software Toolchain"
+description: "The software toolchain includes static analyzers to check assertions about your program; optimizing compilers to translate your program to machine language; operating systems and libraries to supply context for your program. The Verified Software Toolchain project assures with machine-checked proofs that the assertions claimed at the top of the toolchain really hold in the machine-language program, running in the operating-system context."
```

Comment:
Not a requirement, but any chance we could format the `description` content with proper linebreaks? The ideal would that every line in the opam file is at most 80 characters.

Reply:
I had a look at this and I must say that I find it better without explicit breaks. This way opam wraps the lines according to the width of the terminal. If I put in manual breaks this would not work any more. I am using `opam show coq-vst` to check this.

Under what circumstances does this produce suboptimal output?

Reply:
Ah sorry - you are after the opam file - not the user output?

My question is then how to have line breaks in the opam file without having annoying line breaks in `opam show`.

Reply:
Yes, I meant introducing linebreaks in the actual `opam` file to come below the 80 character limit (as is conventional). I typically have a very wide terminal, and when I do `opam show` a description without manual linebreaks will flow across the entire width of the screen, lowering readability.